### ISSUE TITLE
Mark messages as read on mobile devices

### ISSUE THREAD

**Issue by CodeAlDente** [RainLoop/rainloop-webmail#1730]
The "more" button does not show up on mobile devices. As a result I have to open every single message to mark it as read. Please provide the option to mark messages as read / unread on mobile devices.

**Comment by nomadturk** [comment]
I second this.
You can not mark messages on mobile theme due to the lack of "more"

**Comment by prguru** [comment]
Also missing this feature in the **mobile first** world. Of course all of the features must show up in the mobile view...

**Comment by numeropi** [comment]
Yeah, it would be great to have that icon in the mobile view... for the moment the only solution is to change to Desktop view trough the User profile icon and do it and switch back again. Thanks for the great work.
